sway-lsp/src/server.rs

```diff
@@ -70,24 +72,27 @@ fn capabilities() -> ServerCapabilities {
 }
 
 impl Backend {
-    async fn publish_diagnostics(&self, uri: Url, diagnostics: Vec<Diagnostic>) {
+    async fn publish_diagnostics(
+        &self,
+        uri: &Url,
+        diagnostics: Vec<Diagnostic>,
+        token_map: &TokenMap,
+    ) {
         // If parsed_tokens_as_warnings is true, take over the normal error and warning display behavior
         // and instead show the parsed tokens as warnings.
         // This is useful for debugging the lsp parser.
         if self.config.parsed_tokens_as_warnings {
-            if let Some(document) = self.session.documents.get(uri.path()) {
-                let diagnostics = debug::generate_warnings_for_parsed_tokens(document.token_map());
+            //let diagnostics = debug::generate_warnings_for_parsed_tokens(token_map);
```

Comment:
Left over from debugging?

Reply:
I actually had the wrong line commented out, thanks for this.